project/frontend/src/components/MapContainer.js

```diff
@@ -65,30 +67,30 @@ function MapContainer(props) {
     const { restaurantName, percentMatch } = props;
     const infoWindowStyle = {
       position: 'relative',
-      bottom: 80,
-      left: '-45px',
-      width: 220,
+      width: '344px',
+      fontFamily: 'Montserrat',
+      transform: 'translate(calc(-50% + 15px), calc(-100% - 38px))',
       backgroundColor: 'white',
       boxShadow: '0 2px 7px 1px rgba(0, 0, 0, 0.3)',
-      padding: 10,
-      fontSize: 14,
-      zIndex: 100,
+      textAlign: 'left',
+      padding: '16px',
+      zIndex: '50',
+      borderRadius: '4px',
     };
 
     return (
-      <div style={infoWindowStyle}>
-        <div style={{ fontSize: 16 }}>
-          <span style={{ float: 'right' }}>{percentMatch}</span>
-          {restaurantName}
-        </div>
+      <div className='restaurant-header' style={infoWindowStyle}>
+        <h5 className={'restaurant-name'}>{restaurantName}</h5>
+        <h5 className='restaurant-score'>{percentMatch}</h5>
```

Comment:
is it a convention to add classNames to everything? I'm just wondering if we need them if we don't have a css file for this

Reply:
The class names do actually reference a class I have defined in `RestaurantCard.css` so that they match the header in the card.

Reply:
Since this map is always being used with the cards nearby I figured it was safe to assume that `RestaurantCard.css` would be loaded.

Reply:
so you don't have to import `RestaurantCard.css` into `MapContainer.js` in order for the styling to work?

Reply:
No not explicitly. When you import a stylesheet react just appends the entire sheet's content inside the <head> element of the html page, so when it was imported in RestaurantCard.js it got added to <head> and then when this map renders it finds that css class in the <head> element.